### ISSUE TITLE
Don't show welcome dialog if using shell to open .gfxtrace file

### ISSUE THREAD

**Issue by ben-clayton** [google/gapid#1513]
At least on macOS, if you double click a `.gfxtrace` file in finder, you'll have GAPID launch (typically without focus?) and show you the welcome screen (Open File, Open Recent, Capture) while the trace loads in the background.

We should make sure the client comes to focus (it'll often silently open in a different macOS 'desktop' for me) and hide the welcome screen.

**Comment by pmuetschard** [comment]
Yes this is macOS only, since it doesn't actually pass the file to be opened as a command line parameter, but via a UI event.
